### PR TITLE
Feature/8776 move item history to dialog

### PR DESCRIPTION
**Pull request by amoedoamorim**

### CODE REVIEW

File: src/app/components/media/ItemHistoryDialog.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      <DialogContent>
+        <MediaLog
+          media={projectMedia}
+          team={team}
+        />
+      </DialogContent>
+    </Dialog>
+  );
+};
+
```
Comment (caiosba) [inline src/app/components/media/ItemHistoryDialog.js:49]: Please add the `PropTypes`.

**Review by caiosba — CHANGES_REQUESTED** [review]
Looks good to me - just one thing: please add `PropTypes` to the `ItemHistoryDialog` component.

File: src/app/components/media/ItemHistoryDialog.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ItemHistoryDialog.propTypes = {
+  open: PropTypes.bool.isRequired,
+  projectMedia: PropTypes.object.isRequired,
+  team: PropTypes.object.isRequired,
```
Comment (amoedoamorim) [inline src/app/components/media/ItemHistoryDialog.js:54]: I was wondering, in cases where the prop is just received to be passed down, is it okay to now specify what's the shape of the prop? Then specify it where the shape is actually needed?

Comment (caiosba) [reply src/app/components/media/ItemHistoryDialog.js:54]: Yes, that's what I do... I specify only as `object` and then at the final "destination" I specify the actual shape.

**Pull request merged**